network/p2p/ATTESTER.md

```diff
@@ -0,0 +1,74 @@
+# Attester message validation
+
+## Message reactions
+
+Node can react to a message in three ways:
+- `Accept` - message is valid, accepted and processed further.
+- `Ignore` - message is invalid, ignored and not processed further, violation is noted.
+- `Reject` - message is rejected, ignored and not processed further, peer is banned.
+
+## Attester assertions
```

Comment:
we need to add to each assertion, what is the action if it fails it

Reply:
@lior-blox Added, but please review because I'm not sure about correctness